.github/workflows/ci.yml

```diff
@@ -19,12 +19,17 @@ jobs:
     steps:
 
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+
+      - name: Setup NodeJS
```

Comment:
Oh, this is probably not needed as we are precompiling assets in the Capistrano deployment tasks.